Add an example of default_hash (#122)
logstash-plugins/logstash-integration-jdbc: docs/filter-jdbc_static.asciidoc

follow the positional ordering.
<5> Specifies the event field that will store the looked-up data. If the lookup
returns multiple columns, the data is stored as a JSON object within the field.

> <6> Creates a default user if the user can not be found in the users database. 

Reviewer: ```suggestion
<6> Uses values for user fields if the user can not be found in the users database. 
```

> In which case the <<plugins-{type}s-{plugin}-tag_on_default_use>> will be called.

Reviewer: ```suggestion
In which case the <<plugins-{type}s-{plugin}-tag_on_default_use>> tags list is used to mark the event.
```

follow the positional ordering.
<5> Specifies the event field that will store the looked-up data. If the lookup
returns multiple columns, the data is stored as a JSON object within the field.

> <6> Uses values for user fields if the user can not be found in the users database. 
> In which case the <<plugins-{type}s-{plugin}-tag_on_default_use>> tags list is used to mark the event.

Reviewer 2: ```suggestion
<6> Tags user fields with the format defined in <<plugins-{type}s-{plugin}-tag_on_default_use>> when a user is not found in the users database. 
```

Reviewer: I think the meaning it's little bit different.
In case the user is not found in the database, then it still create a new event, but with the data that's provided into `default_hash`, which has some default values matching the columns.
So if a DB lookup result an empty set it brings the data from this "static default values" map, and tags the event with the list present in `tag_on_default_use`.

Author: Ha right. I should not have commited the proposal... :(

Author: I reverted my last commit

Reviewer: Expect @karenzone come up with a nice and fluent sentence to express that concept and avoid the previous tongue twister

> In which case the <<plugins-{type}s-{plugin}-tag_on_default_use>> tags list is used to mark the event.

Reviewer: To me this rephrase seems really good and smooth.

@karenzone 
>  Where does the "static default values" map get populated.

It's populate reading the `default_hash` setting in the [local_lookups](https://www.elastic.co/guide/en/logstash/8.6/plugins-filters-jdbc_static.html#plugins-filters-jdbc_static-local_lookups)

> In which case the <<plugins-{type}s-{plugin}-tag_on_default_use>> tags list is used to mark the event.

Reviewer 2: I tweaked it again to provide more context to the `default_hash` setting to clearly explain the way those two settings work together.
